{
    "data": [
        {
            "test_name": "MantidQtWidgetsInstrumentWidgetTestQt5_InstrumentWidgetTest",
            "latest_failed_build": "1989",
            "agent_name": "Linux",
            "last_failed_outcome": "Failed",
            "pipeline_name": "pull_requests-conda-linux",
            "problem_count": 2,
            "past_outcome": {
                "build_keys": [
                    "1991",
                    "1990",
                    "1989",
                    "1988",
                    "1987",
                    "1986",
                    "1985",
                    "1984",
                    "1983",
                    "1982",
                    "1981",
                    "1980",
                    "1979",
                    "1978",
                    "1977",
                    "1976",
                    "1975",
                    "1974",
                    "1973",
                    "1972",
                    "1971",
                    "1970",
                    "1969",
                    "1968",
                    "1967",
                    "1966",
                    "1965",
                    "1964",
                    "1963",
                    "1962"
                ],
                "past_outcome_list": [
                    "None",
                    "None",
                    "Failed",
                    "None",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Failed",
                    "Passed",
                    "None",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "None",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed"
                ]
            },
            "last_stack_traces": {
                "test_name": "MantidQtWidgetsInstrumentWidgetTestQt5_InstrumentWidgetTest",
                "build": "1989",
                "agent_name": "Linux",
                "trial": 3,
                "outcome": "Failed",
                "test_time": "3.48",
                "stacktrace": "FrameworkManager-[Notice] Welcome to Mantid 6.5.20221121.1003\nFrameworkManager-[Notice] Please cite: http://dx.doi.org/10.1016/j.nima.2014.07.029 and this release: http://dx.doi.org/10.5286/Software/Mantid\nRunning 10 testsQStandardPaths: XDG_RUNTIME_DIR not set, defaulting to &#x27;/tmp/runtime-abc&#x27;\nlibEGL warning: DRI2: failed to open swrast (search paths /usr/lib64/dri)\nlibEGL warning: DRI2: failed to open swrast (search paths /usr/lib64/dri)\nlibEGL warning: DRI2: failed to open swrast (search paths /usr/lib64/dri)\nlibEGL warning: DRI2: failed to open swrast (search paths /usr/lib64/dri)\nQXcbIntegration: Cannot create platform OpenGL context, neither GLX nor EGL are enabled\nQXcbIntegration: Cannot create platform OpenGL context, neither GLX nor EGL are enabled\nQObject::connect: Cannot connect (null)::colorMapChanged() to MantidQt::MantidWidgets::ProjectionSurface::colorMapChanged()\nQObject::connect: Cannot connect (null)::refreshView() to MantidQt::MantidWidgets::ProjectionSurface::refreshView()\nQObject::connect: Cannot queue arguments of type &#x27;std::string&#x27;\n(Make sure &#x27;std::string&#x27; is registered using qRegisterMetaType().)\nQObject::connect: Cannot queue arguments of type &#x27;std::string&#x27;\n(Make sure &#x27;std::string&#x27; is registered using qRegisterMetaType().)\nQObject::connect: Cannot queue arguments of type &#x27;std::string&#x27;\n(Make sure &#x27;std::string&#x27; is registered using qRegisterMetaType().)\nQXcbIntegration: Cannot create platform OpenGL context, neither GLX nor EGL are enabled\nQObject::connect: Cannot connect (null)::colorMapChanged() to MantidQt::MantidWidgets::ProjectionSurface::colorMapChanged()\nQObject::connect: Cannot connect (null)::refreshView() to MantidQt::MantidWidgets::ProjectionSurface::refreshView()\nunknown file: Failure\n\nUnexpected mock function call - returning directly.\n    Function call: connect(0x55d4efac1620, 0x7fad66dd4aaf pointing to &quot;2initWidget(bool, bool)&quot;, 0x7ffeaaa52630, 0x7fad66dd4a97 pointing to &quot;1initWidget(bool, bool)&quot;)\nGoogle Mock tried the following 11 expectations, but none matched:\n\n/jenkins_workdir/workspace/pull_requests-conda-linux/qt/widgets/instrumentview/test/InstrumentWidget/InstrumentWidgetTest.h:308: tried expectation #0: EXPECT_CALL(mock, connect(_, StrEq(signal), _, StrEq(slot)))...\n  Expected arg #1: is equal to &quot;2enableLighting(bool)&quot;\n           Actual: 0x7fad66dd4aaf pointing to &quot;2initWidget(bool, bool)&quot;\n  Expected arg #3: is equal to &quot;1enableLighting(bool)&quot;\n           Actual: 0x7fad66dd4a97 pointing to &quot;1initWidget(bool, bool)&quot;\n         Expected: to be called once\n           Actual: called once - saturated and active\n/jenkins_workdir/workspace/pull_requests-conda-linux/qt/widgets/instrumentview/test/InstrumentWidget/InstrumentWidgetTest.h:308: tried expectation #1: EXPECT_CALL(mock, connect(_, StrEq(signal), _, StrEq(slot)))...\n  Expected arg #1: is equal to &quot;2changed(double, double)&quot;\n           Actual: 0x7fad66dd4aaf pointing to &quot;2initWidget(bool, bool)&quot;\n  Expected arg #3: is equal to &quot;1setIntegrationRange(double, double)&quot;\n           Actual: 0x7fad66dd4a97 pointing to &quot;1initWidget(bool, bool)&quot;\n         Expected: to be called once\n           Actual: called once - saturated and active\n/jenkins_workdir/workspace/pull_requests-conda-linux/qt/widgets/instrumentview/test/InstrumentWidget/InstrumentWidgetTest.h:308: tried expectation #2: EXPECT_CALL(mock, connect(_, StrEq(signal), _, StrEq(slot)))...\n  Expected arg #1: is equal to &quot;2clicked()&quot;\n           Actual: 0x7fad66dd4aaf pointing to &quot;2initWidget(bool, bool)&quot;\n  Expected arg #3: is equal to &quot;1helpClicked()&quot;\n           Actual: 0x7fad66dd4a97 pointing to &quot;1initWidget(bool, bool)&quot;\n         Expected: to be called once\n           Actual: called once - saturated and active\n/jenkins_workdir/workspace/pull_requests-conda-linux/qt/widgets/instrumentview/test/InstrumentWidget/InstrumentWidgetTest.h:308: tried expectation #3: EXPECT_CALL(mock, connect(_, StrEq(signal), _, StrEq(slot)))...\n  Expected arg #1: is equal to &quot;2setAutoscaling(bool)&quot;\n           Actual: 0x7fad66dd4aaf pointing to &quot;2initWidget(bool, bool)&quot;\n  Expected arg #3: is equal to &quot;1setColorMapAutoscaling(bool)&quot;\n           Actual: 0x7fad66dd4a97 pointing to &quot;1initWidget(bool, bool)&quot;\n         Expected: to be called once\n           Actual: never called - unsatisfied and active\n/jenkins_workdir/workspace/pull_requests-conda-linux/qt/widgets/instrumentview/test/InstrumentWidget/InstrumentWidgetTest.h:308: tried expectation #4: EXPECT_CALL(mock, connect(_, StrEq(signal), _, StrEq(slot)))...\n  Expected arg #1: is equal to &quot;2rescaleColorMap()&quot;\n           Actual: 0x7fad66dd4aaf pointing to &quot;2initWidget(bool, bool)&quot;\n  Expected arg #3: is equal to &quot;1setupColorMap()&quot;\n           Actual: 0x7fad66dd4a97 pointing to &quot;1initWidget(bool, bool)&quot;\n         Expected: to be called once\n           Actual: never called - unsatisfied and active\n/jenkins_workdir/workspace/pull_requests-conda-linux/qt/widgets/instrumentview/test/InstrumentWidget/InstrumentWidgetTest.h:308: tried expectation #5: EXPECT_CALL(mock, connect(_, StrEq(signal), _, StrEq(slot)))...\n  Expected arg #1: is equal to &quot;2executeAlgorithm(const QString &amp;, const QString &amp;)&quot;\n           Actual: 0x7fad66dd4aaf pointing to &quot;2initWidget(bool, bool)&quot;\n  Expected arg #3: is equal to &quot;1executeAlgorithm(const QString &amp;, const QString &amp;)&quot;\n           Actual: 0x7fad66dd4a97 pointing to &quot;1initWidget(bool, bool)&quot;\n         Expected: to be called once\n           Actual: never called - unsatisfied and active\n/jenkins_workdir/workspace/pull_requests-conda-linux/qt/widgets/instrumentview/test/InstrumentWidget/InstrumentWidgetTest.h:308: tried expectation #6: EXPECT_CALL(mock, connect(_, StrEq(signal), _, StrEq(slot)))...\n  Expected arg #1: is equal to &quot;2changed(double, double)&quot;\n           Actual: 0x7fad66dd4aaf pointing to &quot;2initWidget(bool, bool)&quot;\n  Expected arg #3: is equal to &quot;1changedIntegrationRange(double, double)&quot;\n           Actual: 0x7fad66dd4a97 pointing to &quot;1initWidget(bool, bool)&quot;\n         Expected: to be called once\n           Actual: never called - unsatisfied and active\n/jenkins_workdir/workspace/pull_requests-conda-linux/qt/widgets/instrumentview/test/InstrumentWidget/InstrumentWidgetTest.h:308: tried expectation #7: EXPECT_CALL(mock, connect(_, StrEq(signal), _, StrEq(slot)))...\n  Expected arg #1: is equal to &quot;2currentChanged(int)&quot;\n           Actual: 0x7fad66dd4aaf pointing to &quot;2initWidget(bool, bool)&quot;\n  Expected arg #3: is equal to &quot;1tabChanged(int)&quot;\n           Actual: 0x7fad66dd4a97 pointing to &quot;1initWidget(bool, bool)&quot;\n         Expected: to be called once\n           Actual: never called - unsatisfied and active\n/jenkins_workdir/workspace/pull_requests-conda-linux/qt/widgets/instrumentview/test/InstrumentWidget/InstrumentWidgetTest.h:308: tried expectation #8: EXPECT_CALL(mock, connect(_, StrEq(signal), _, StrEq(slot)))...\n  Expected arg #1: is equal to &quot;2triggered()&quot;\n           Actual: 0x7fad66dd4aaf pointing to &quot;2initWidget(bool, bool)&quot;\n  Expected arg #3: is equal to &quot;1clearPeakOverlays()&quot;\n           Actual: 0x7fad66dd4a97 pointing to &quot;1initWidget(bool, bool)&quot;\n         Expected: to be called once\n           Actual: never called - unsatisfied and active\n/jenkins_workdir/workspace/pull_requests-conda-linux/qt/widgets/instrumentview/test/InstrumentWidget/InstrumentWidgetTest.h:308: tried expectation #9: EXPECT_CALL(mock, connect(_, StrEq(signal), _, StrEq(slot)))...\n  Expected arg #1: is equal to &quot;2triggered()&quot;\n           Actual: 0x7fad66dd4aaf pointing to &quot;2initWidget(bool, bool)&quot;\n  Expected arg #3: is equal to &quot;1clearAlignmentPlane()&quot;\n           Actual: 0x7fad66dd4a97 pointing to &quot;1initWidget(bool, bool)&quot;\n         Expected: to be called once\n           Actual: never called - unsatisfied and active\n/jenkins_workdir/workspace/pull_requests-conda-linux/qt/widgets/instrumentview/test/InstrumentWidget/InstrumentWidgetTest.h:308: tried expectation #10: EXPECT_CALL(mock, connect(_, StrEq(signal), _, StrEq(slot)))...\n  Expected arg #1: is equal to &quot;2executeAlgorithm(Mantid::API::IAlgorithm_sptr)&quot;\n           Actual: 0x7fad66dd4aaf pointing to &quot;2initWidget(bool, bool)&quot;\n  Expected arg #3: is equal to &quot;1executeAlgorithm(Mantid::API::IAlgorithm_sptr)&quot;\n           Actual: 0x7fad66dd4a97 pointing to &quot;1initWidget(bool, bool)&quot;\n         Expected: to be called once\n           Actual: never called - unsatisfied and active\n\n********* UNHANDLED EXCEPTION *********\n  what(): unknown file: Failure\n\nUnexpected mock function call - returning directly.\n    Function call: connect(0x55d4efac1620, 0x7fad66dd4aaf pointing to &quot;2initWidget(bool, bool)&quot;, 0x7ffeaaa52630, 0x7fad66dd4a97 pointing to &quot;1initWidget(bool, bool)&quot;)\nGoogle Mock tried the following 11 expectations, but none matched:\n\n/jenkins_workdir/workspace/pull_requests-conda-linux/qt/widgets/instrumentview/test/InstrumentWidget/InstrumentWidgetTest.h:308: tried expectation #0: EXPECT_CALL(mock, connect(_, StrEq(signal), _, StrEq(slot)))...\n  Expected arg #1: is equal to &quot;2enableLighting(bool)&quot;\n           Actual: 0x7fad66dd4aaf pointing to &quot;2initWidget(bool, bool)&quot;\n  Expected arg #3: is equal to &quot;1enableLighting(bool)&quot;\n           Actual: 0x7fad66dd4a97 pointing to &quot;1initWidget(bool, bool)&quot;\n         Expected: to be called once\n           Actual: called once - saturated and active\n/jenkins_workdir/workspace/pull_requests-conda-linux/qt/widgets/instrumentview/test/InstrumentWidget/InstrumentWidgetTest.h:308: tried expectation #1: EXPECT_CALL(mock, connect(_, StrEq(signal), _, StrEq(slot)))...\n  Expected arg #1: is equal to &quot;2changed(double, double)&quot;\n           Actual: 0x7fad66dd4aaf pointing to &quot;2initWidget(bool, bool)&quot;\n  Expected arg #3: is equal to &quot;1setIntegrationRange(double, double)&quot;\n           Actual: 0x7fad66dd4a97 pointing to &quot;1initWidget(bool, bool)&quot;\n         Expected: to be called once\n           Actual: called once - saturated and active\n/jenkins_workdir/workspace/pull_requests-conda-linux/qt/widgets/instrumentview/test/InstrumentWidget/InstrumentWidgetTest.h:308: tried expectation #2: EXPECT_CALL(mock, connect(_, StrEq(signal), _, StrEq(slot)))...\n  Expected arg #1: is equal to &quot;2clicked()&quot;\n           Actual: 0x7fad66dd4aaf pointing to &quot;2initWidget(bool, bool)&quot;\n  Expected arg #3: is equal to &quot;1helpClicked()&quot;\n           Actual: 0x7fad66dd4a97 pointing to &quot;1initWidget(bool, bool)&quot;\n         Expected: to be called once\n           Actual: called once - saturated and active\n/jenkins_workdir/workspace/pull_requests-conda-linux/qt/widgets/instrumentview/test/InstrumentWidget/InstrumentWidgetTest.h:308: tried expectation #3: EXPECT_CALL(mock, connect(_, StrEq(signal), _, StrEq(slot)))...\n  Expected arg #1: is equal to &quot;2setAutoscaling(bool)&quot;\n           Actual: 0x7fad66dd4aaf pointing to &quot;2initWidget(bool, bool)&quot;\n  Expected arg #3: is equal to &quot;1setColorMapAutoscaling(bool)&quot;\n           Actual: 0x7fad66dd4a97 pointing to &quot;1initWidget(bool, bool)&quot;\n         Expected: to be called once\n           Actual: never called - unsatisfied and active\n/jenkins_workdir/workspace/pull_requests-conda-linux/qt/widgets/instrumentview/test/InstrumentWidget/InstrumentWidgetTest.h:308: tried expectation #4: EXPECT_CALL(mock, connect(_, StrEq(signal), _, StrEq(slot)))...\n  Expected arg #1: is equal to &quot;2rescaleColorMap()&quot;\n           Actual: 0x7fad66dd4aaf pointing to &quot;2initWidget(bool, bool)&quot;\n  Expected arg #3: is equal to &quot;1setupColorMap()&quot;\n           Actual: 0x7fad66dd4a97 pointing to &quot;1initWidget(bool, bool)&quot;\n         Expected: to be called once\n           Actual: never called - unsatisfied and active\n/jenkins_workdir/workspace/pull_requests-conda-linux/qt/widgets/instrumentview/test/InstrumentWidget/InstrumentWidgetTest.h:308: tried expectation #5: EXPECT_CALL(mock, connect(_, StrEq(signal), _, StrEq(slot)))...\n  Expected arg #1: is equal to &quot;2executeAlgorithm(const QString &amp;, const QString &amp;)&quot;\n           Actual: 0x7fad66dd4aaf pointing to &quot;2initWidget(bool, bool)&quot;\n  Expected arg #3: is equal to &quot;1executeAlgorithm(const QString &amp;, const QString &amp;)&quot;\n           Actual: 0x7fad66dd4a97 pointing to &quot;1initWidget(bool, bool)&quot;\n         Expected: to be called once\n           Actual: never called - unsatisfied and active\n/jenkins_workdir/workspace/pull_requests-conda-linux/qt/widgets/instrumentview/test/InstrumentWidget/InstrumentWidgetTest.h:308: tried expectation #6: EXPECT_CALL(mock, connect(_, StrEq(signal), _, StrEq(slot)))...\n  Expected arg #1: is equal to &quot;2changed(double, double)&quot;\n           Actual: 0x7fad66dd4aaf pointing to &quot;2initWidget(bool, bool)&quot;\n  Expected arg #3: is equal to &quot;1changedIntegrationRange(double, double)&quot;\n           Actual: 0x7fad66dd4a97 pointing to &quot;1initWidget(bool, bool)&quot;\n         Expected: to be called once\n           Actual: never called - unsatisfied and active\n/jenkins_workdir/workspace/pull_requests-conda-linux/qt/widgets/instrumentview/test/InstrumentWidget/InstrumentWidgetTest.h:308: tried expectation #7: EXPECT_CALL(mock, connect(_, StrEq(signal), _, StrEq(slot)))...\n  Expected arg #1: is equal to &quot;2currentChanged(int)&quot;\n           Actual: 0x7fad66dd4aaf pointing to &quot;2initWidget(bool, bool)&quot;\n  Expected arg #3: is equal to &quot;1tabChanged(int)&quot;\n           Actual: 0x7fad66dd4a97 pointing to &quot;1initWidget(bool, bool)&quot;\n         Expected: to be called once\n           Actual: never called - unsatisfied and active\n/jenkins_workdir/workspace/pull_requests-conda-linux/qt/widgets/instrumentview/test/InstrumentWidget/InstrumentWidgetTest.h:308: tried expectation #8: EXPECT_CALL(mock, connect(_, StrEq(signal), _, StrEq(slot)))...\n  Expected arg #1: is equal to &quot;2triggered()&quot;\n           Actual: 0x7fad66dd4aaf pointing to &quot;2initWidget(bool, bool)&quot;\n  Expected arg #3: is equal to &quot;1clearPeakOverlays()&quot;\n           Actual: 0x7fad66dd4a97 pointing to &quot;1initWidget(bool, bool)&quot;\n         Expected: to be called once\n           Actual: never called - unsatisfied and active\n/jenkins_workdir/workspace/pull_requests-conda-linux/qt/widgets/instrumentview/test/InstrumentWidget/InstrumentWidgetTest.h:308: tried expectation #9: EXPECT_CALL(mock, connect(_, StrEq(signal), _, StrEq(slot)))...\n  Expected arg #1: is equal to &quot;2triggered()&quot;\n           Actual: 0x7fad66dd4aaf pointing to &quot;2initWidget(bool, bool)&quot;\n  Expected arg #3: is equal to &quot;1clearAlignmentPlane()&quot;\n           Actual: 0x7fad66dd4a97 pointing to &quot;1initWidget(bool, bool)&quot;\n         Expected: to be called once\n           Actual: never called - unsatisfied and active\n/jenkins_workdir/workspace/pull_requests-conda-linux/qt/widgets/instrumentview/test/InstrumentWidget/InstrumentWidgetTest.h:308: tried expectation #10: EXPECT_CALL(mock, connect(_, StrEq(signal), _, StrEq(slot)))...\n  Expected arg #1: is equal to &quot;2executeAlgorithm(Mantid::API::IAlgorithm_sptr)&quot;\n           Actual: 0x7fad66dd4aaf pointing to &quot;2initWidget(bool, bool)&quot;\n  Expected arg #3: is equal to &quot;1executeAlgorithm(Mantid::API::IAlgorithm_sptr)&quot;\n           Actual: 0x7fad66dd4a97 pointing to &quot;1initWidget(bool, bool)&quot;\n         Expected: to be called once\n           Actual: never called - unsatisfied and active\n\nBacktrace:\n /jenkins_workdir/workspace/pull_requests-conda-linux/build/bin/libMantidAPI.so(+0x1f113a) [0x7fad646be13a]\n /jenkins_workdir/workspace/pull_requests-conda-linux/build/bin/libMantidAPI.so(+0x100fcd) [0x7fad645cdfcd]\n /jenkins_workdir/workspace/pull_requests-conda-linux/mambaforge/envs/mantid-developer/lib/libstdc++.so.6(+0xb0514) [0x7fad63883514]\n /jenkins_workdir/workspace/pull_requests-conda-linux/mambaforge/envs/mantid-developer/lib/libstdc++.so.6(+0xb0566) [0x7fad63883566]\n /jenkins_workdir/workspace/pull_requests-conda-linux/mambaforge/envs/mantid-developer/lib/libstdc++.so.6(__cxa_rethrow+0) [0x7fad63883758]\n /jenkins_workdir/workspace/pull_requests-conda-linux/build/bin/MantidQtWidgetsInstrumentWidgetTestQt5(+0x37ebf) [0x55d4ede04ebf]\n /jenkins_workdir/workspace/pull_requests-conda-linux/build/bin/MantidQtWidgetsInstrumentWidgetTestQt5(+0xe78a9) [0x55d4edeb48a9]\n /jenkins_workdir/workspace/pull_requests-conda-linux/build/bin/MantidQtWidgetsInstrumentWidgetTestQt5(+0xbcf64) [0x55d4ede89f64]\n /jenkins_workdir/workspace/pull_requests-conda-linux/build/bin/MantidQtWidgetsInstrumentWidgetTestQt5(+0xc20d2) [0x55d4ede8f0d2]\n /jenkins_workdir/workspace/pull_requests-conda-linux/build/bin/MantidQtWidgetsInstrumentWidgetTestQt5(+0x5e3c4) [0x55d4ede2b3c4]\n /jenkins_workdir/workspace/pull_requests-conda-linux/build/bin/libMantidQtWidgetsInstrumentViewQt5.so(_ZN8MantidQt13MantidWidgets16InstrumentWidget20resetInstrumentActorEbbddb+0x593) [0x7fad66d4cf33]\n /jenkins_workdir/workspace/pull_requests-conda-linux/build/bin/libMantidQtWidgetsInstrumentViewQt5.so(_ZN8MantidQt13MantidWidgets16InstrumentWidgetC1E7QStringP7QWidgetbbddbbNS0_6Detail12DependenciesE+0x789) [0x7fad66d54509]\n /jenkins_workdir/workspace/pull_requests-conda-linux/build/bin/MantidQtWidgetsInstrumentWidgetTestQt5(+0x663a9) [0x55d4ede333a9]\n /jenkins_workdir/workspace/pull_requests-conda-linux/build/bin/MantidQtWidgetsInstrumentWidgetTestQt5(+0x9144b) [0x55d4ede5e44b]\n /jenkins_workdir/workspace/pull_requests-conda-linux/build/bin/MantidQtWidgetsInstrumentWidgetTestQt5(+0xa45d3) [0x55d4ede715d3]\n /jenkins_workdir/workspace/pull_requests-conda-linux/build/bin/MantidQtWidgetsInstrumentWidgetTestQt5(+0x38d47) [0x55d4ede05d47]\n /lib64/libc.so.6(__libc_start_main+0xf5) [0x7fad6310b555]\n /jenkins_workdir/workspace/pull_requests-conda-linux/build/bin/MantidQtWidgetsInstrumentWidgetTestQt5(+0x39fd9) [0x55d4ede06fd9]\n\n[ FATAL ] /jenkins_workdir/workspace/pull_requests-conda-linux/build/_deps/googletest-src/googletest/include/gtest/internal/gtest-port.h:1670:: pthread_mutex_destroy(&amp;mutex_)failed with error 16\nSubprocess aborted\n\n          Start  931: DataHandlingTest_LoadSESANSTest\n"
            }
        },
        {
            "test_name": "python.algorithms.MaskBTPTest.MaskBTPTest",
            "latest_failed_build": "1975",
            "agent_name": "Linux",
            "last_failed_outcome": "Failed",
            "pipeline_name": "pull_requests-conda-linux",
            "problem_count": 2,
            "past_outcome": {
                "build_keys": [
                    "1991",
                    "1990",
                    "1989",
                    "1988",
                    "1987",
                    "1986",
                    "1985",
                    "1984",
                    "1983",
                    "1982",
                    "1981",
                    "1980",
                    "1979",
                    "1978",
                    "1977",
                    "1976",
                    "1975",
                    "1974",
                    "1973",
                    "1972",
                    "1971",
                    "1970",
                    "1969",
                    "1968",
                    "1967",
                    "1966",
                    "1965",
                    "1964",
                    "1963",
                    "1962"
                ],
                "past_outcome_list": [
                    "None",
                    "None",
                    "Passed",
                    "None",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "None",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Failed",
                    "Failed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "None",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed"
                ]
            },
            "last_stack_traces": {
                "test_name": "python.algorithms.MaskBTPTest.MaskBTPTest",
                "build": "1975",
                "agent_name": "Linux",
                "trial": 3,
                "outcome": "Failed",
                "test_time": "46.82",
                "stacktrace": "FrameworkManager-[Notice] Welcome to Mantid 6.5.20221121.1628\nFrameworkManager-[Notice] Please cite: http://dx.doi.org/10.1016/j.nima.2014.07.029 and this release: http://dx.doi.org/10.5286/Software/Mantid\nLoadEmptyInstrument-[Notice] LoadEmptyInstrument started\nLoadEmptyInstrument-[Notice] LoadEmptyInstrument successful, Duration 0.74 seconds\nMaskBTP-[Notice] MaskBTP started\nMaskBTP-[Notice] MaskBTP successful, Duration 0.07 seconds\nMaskBTP-[Notice] MaskBTP started\nMaskBTP-[Notice] MaskBTP successful, Duration 0.05 seconds\nMaskBTP-[Notice] MaskBTP started\nMaskBTP-[Notice] MaskBTP successful, Duration 0.15 seconds\n.LoadEmptyInstrument-[Notice] LoadEmptyInstrument started\nLoadEmptyInstrument-[Notice] LoadEmptyInstrument successful, Duration 0.16 seconds\nMaskBTP-[Notice] MaskBTP started\nMaskBTP-[Notice] MaskBTP successful, Duration 0.03 seconds\nMaskBTP-[Notice] MaskBTP started\nMaskBTP-[Notice] MaskBTP successful, Duration 0.04 seconds\nMaskBTP-[Notice] MaskBTP started\nMaskBTP-[Notice] MaskBTP successful, Duration 0.06 seconds\n.MaskBTP-[Notice] MaskBTP started\nLoadEmptyInstrument-[Notice] LoadEmptyInstrument started (child)\nLoadEmptyInstrument-[Notice] LoadEmptyInstrument successful, Duration 12.93 seconds\nMaskBTP-[Notice] MaskBTP successful, Duration 14.68 seconds\nFMaskBTP-[Notice] MaskBTP started\nLoadEmptyInstrument-[Notice] LoadEmptyInstrument started (child)\nLoadEmptyInstrument-[Notice] LoadEmptyInstrument successful, Duration 0.39 seconds\nMaskBTP-[Notice] MaskBTP successful, Duration 0.46 seconds\nMaskBTP-[Notice] MaskBTP started\nMaskBTP-[Notice] MaskBTP successful, Duration 0.10 seconds\nMaskBTP-[Notice] MaskBTP started\nMaskBTP-[Notice] MaskBTP successful, Duration 0.03 seconds\nDeleteWorkspace-[Notice] DeleteWorkspace started\nDeleteWorkspace-[Notice] DeleteWorkspace successful, Duration 0.00 seconds\n.MaskBTP-[Notice] MaskBTP started\nMaskBTP-[Error] Error in execution of algorithm MaskBTP:\nMaskBTP-[Error] Instrument &#x27;testInst&#x27; not in the allowed list\nMaskBTP-[Error]   at line 94 in &#x27;/jenkins_workdir/workspace/pull_requests-conda-linux/Framework/PythonInterface/plugins/algorithms/MaskBTP.py&#x27;\nDeleteWorkspace-[Notice] DeleteWorkspace started\nDeleteWorkspace-[Notice] DeleteWorkspace successful, Duration 0.00 seconds\n.MaskBTP-[Notice] MaskBTP started\nLoadEmptyInstrument-[Notice] LoadEmptyInstrument started (child)\nLoadEmptyInstrument-[Notice] LoadEmptyInstrument successful, Duration 1.37 seconds\nMaskBTP-[Error] Error in execution of algorithm MaskBTP:\nMaskBTP-[Error] None of the indices ([128]) are in range\nMaskBTP-[Error]   at line 134 in &#x27;/jenkins_workdir/workspace/pull_requests-conda-linux/Framework/PythonInterface/plugins/algorithms/MaskBTP.py&#x27;\nMaskBTP-[Error]   caused by line 177 in &#x27;/jenkins_workdir/workspace/pull_requests-conda-linux/Framework/PythonInterface/plugins/algorithms/MaskBTP.py&#x27;\nMaskBTP-[Notice] MaskBTP started\nLoadEmptyInstrument-[Notice] LoadEmptyInstrument started (child)\nLoadEmptyInstrument-[Notice] LoadEmptyInstrument successful, Duration 2.12 seconds\nMaskBTP-[Error] Error in execution of algorithm MaskBTP:\nMaskBTP-[Error] Out of range index=1 for SEQUOIA instrument bank numbers\nMaskBTP-[Error]   at line 107 in &#x27;/jenkins_workdir/workspace/pull_requests-conda-linux/Framework/PythonInterface/plugins/algorithms/MaskBTP.py&#x27;\nMaskBTP-[Error]   caused by line 107 in &#x27;/jenkins_workdir/workspace/pull_requests-conda-linux/Framework/PythonInterface/plugins/algorithms/MaskBTP.py&#x27;\nMaskBTP-[Error]   caused by line 199 in &#x27;/jenkins_workdir/workspace/pull_requests-conda-linux/Framework/PythonInterface/plugins/algorithms/MaskBTP.py&#x27;\nMaskBTP-[Notice] MaskBTP started\nLoadEmptyInstrument-[Notice] LoadEmptyInstrument started (child)\nLoadEmptyInstrument-[Notice] LoadEmptyInstrument successful, Duration 0.41 seconds\nMaskBTP-[Error] Error in execution of algorithm MaskBTP:\nMaskBTP-[Error] Out of range index=22 for SEQUOIA instrument bank numbers\nMaskBTP-[Error]   at line 107 in &#x27;/jenkins_workdir/workspace/pull_requests-conda-linux/Framework/PythonInterface/plugins/algorithms/MaskBTP.py&#x27;\nMaskBTP-[Error]   caused by line 107 in &#x27;/jenkins_workdir/workspace/pull_requests-conda-linux/Framework/PythonInterface/plugins/algorithms/MaskBTP.py&#x27;\nMaskBTP-[Error]   caused by line 199 in &#x27;/jenkins_workdir/workspace/pull_requests-conda-linux/Framework/PythonInterface/plugins/algorithms/MaskBTP.py&#x27;\nMaskBTP-[Notice] MaskBTP started\nLoadEmptyInstrument-[Notice] LoadEmptyInstrument started (child)\nLoadEmptyInstrument-[Notice] LoadEmptyInstrument successful, Duration 0.18 seconds\nMaskBTP-[Error] Error in execution of algorithm MaskBTP:\nMaskBTP-[Error] None of the indices ([17]) are in range\nMaskBTP-[Error]   at line 132 in &#x27;/jenkins_workdir/workspace/pull_requests-conda-linux/Framework/PythonInterface/plugins/algorithms/MaskBTP.py&#x27;\nMaskBTP-[Error]   caused by line 177 in &#x27;/jenkins_workdir/workspace/pull_requests-conda-linux/Framework/PythonInterface/plugins/algorithms/MaskBTP.py&#x27;\nDeleteWorkspace-[Notice] DeleteWorkspace started\nDeleteWorkspace-[Notice] DeleteWorkspace successful, Duration 0.00 seconds\nDeleteWorkspace-[Notice] DeleteWorkspace started\nDeleteWorkspace-[Notice] DeleteWorkspace successful, Duration 0.01 seconds\nDeleteWorkspace-[Notice] DeleteWorkspace started\nDeleteWorkspace-[Notice] DeleteWorkspace successful, Duration 0.02 seconds\n.MaskBTP-[Notice] MaskBTP started\nLoadEmptyInstrument-[Notice] LoadEmptyInstrument started (child)\nLoadEmptyInstrument-[Notice] LoadEmptyInstrument successful, Duration 0.32 seconds\nMaskBTP-[Notice] MaskBTP successful, Duration 0.43 seconds\nMaskBTP-[Notice] MaskBTP started\nLoadEmptyInstrument-[Notice] LoadEmptyInstrument started (child)\nLoadEmptyInstrument-[Notice] LoadEmptyInstrument successful, Duration 0.21 seconds\nMaskBTP-[Notice] MaskBTP successful, Duration 0.37 seconds\nMaskBTP-[Notice] MaskBTP started\nLoadEmptyInstrument-[Notice] LoadEmptyInstrument started (child)\nLoadEmptyInstrument-[Notice] LoadEmptyInstrument successful, Duration 0.26 seconds\nMaskBTP-[Notice] MaskBTP successful, Duration 0.43 seconds\nMaskBTP-[Notice] MaskBTP started\nLoadEmptyInstrument-[Notice] LoadEmptyInstrument started (child)\nLoadEmptyInstrument-[Notice] LoadEmptyInstrument successful, Duration 0.24 seconds\nMaskBTP-[Notice] MaskBTP successful, Duration 0.38 seconds\nMaskBTP-[Notice] MaskBTP started\nLoadEmptyInstrument-[Notice] LoadEmptyInstrument started (child)\nLoadEmptyInstrument-[Notice] LoadEmptyInstrument successful, Duration 0.20 seconds\nMaskBTP-[Notice] MaskBTP successful, Duration 0.24 seconds\nMaskBTP-[Notice] MaskBTP started\nLoadEmptyInstrument-[Notice] LoadEmptyInstrument started (child)\nLoadEmptyInstrument-[Notice] LoadEmptyInstrument successful, Duration 0.24 seconds\nMaskBTP-[Notice] MaskBTP successful, Duration 0.31 seconds\nMaskBTP-[Notice] MaskBTP started\nLoadEmptyInstrument-[Notice] LoadEmptyInstrument started (child)\nLoadEmptyInstrument-[Notice] LoadEmptyInstrument successful, Duration 0.19 seconds\nMaskBTP-[Notice] MaskBTP successful, Duration 0.27 seconds\nMaskBTP-[Notice] MaskBTP started\nLoadEmptyInstrument-[Notice] LoadEmptyInstrument started (child)\nLoadEmptyInstrument-[Notice] LoadEmptyInstrument successful, Duration 0.29 seconds\nMaskBTP-[Notice] MaskBTP successful, Duration 0.45 seconds\nMaskBTP-[Notice] MaskBTP started\nLoadEmptyInstrument-[Notice] LoadEmptyInstrument started (child)\nLoadEmptyInstrument-[Notice] LoadEmptyInstrument successful, Duration 0.29 seconds\nMaskBTP-[Notice] MaskBTP successful, Duration 0.52 seconds\nMaskBTP-[Notice] MaskBTP started\nLoadEmptyInstrument-[Notice] LoadEmptyInstrument started (child)\nLoadEmptyInstrument-[Notice] LoadEmptyInstrument successful, Duration 0.31 seconds\nMaskBTP-[Notice] MaskBTP successful, Duration 0.55 seconds\nMaskBTP-[Notice] MaskBTP started\nLoadEmptyInstrument-[Notice] LoadEmptyInstrument started (child)\nLoadEmptyInstrument-[Notice] LoadEmptyInstrument successful, Duration 0.36 seconds\nMaskBTP-[Notice] MaskBTP successful, Duration 0.53 seconds\nMaskBTP-[Notice] MaskBTP started\nLoadEmptyInstrument-[Notice] LoadEmptyInstrument started (child)\nLoadEmptyInstrument-[Notice] LoadEmptyInstrument successful, Duration 0.29 seconds\nMaskBTP-[Notice] MaskBTP successful, Duration 0.42 seconds\nMaskBTP-[Notice] MaskBTP started\nLoadEmptyInstrument-[Notice] LoadEmptyInstrument started (child)\nLoadEmptyInstrument-[Notice] LoadEmptyInstrument successful, Duration 0.23 seconds\nMaskBTP-[Notice] MaskBTP successful, Duration 0.31 seconds\nMaskBTP-[Notice] MaskBTP started\nLoadEmptyInstrument-[Notice] LoadEmptyInstrument started (child)\nLoadEmptyInstrument-[Notice] LoadEmptyInstrument successful, Duration 0.16 seconds\nMaskBTP-[Notice] MaskBTP successful, Duration 0.26 seconds\nMaskBTP-[Notice] MaskBTP started\nLoadEmptyInstrument-[Notice] LoadEmptyInstrument started (child)\nLoadEmptyInstrument-[Notice] LoadEmptyInstrument successful, Duration 0.24 seconds\nMaskBTP-[Notice] MaskBTP successful, Duration 0.47 seconds\nMaskBTP-[Notice] MaskBTP started\nLoadEmptyInstrument-[Notice] LoadEmptyInstrument started (child)\nLoadEmptyInstrument-[Notice] LoadEmptyInstrument successful, Duration 0.46 seconds\nMaskBTP-[Notice] MaskBTP successful, Duration 0.66 seconds\nMaskBTP-[Notice] MaskBTP started\nLoadEmptyInstrument-[Notice] LoadEmptyInstrument started (child)\nLoadEmptyInstrument-[Notice] LoadEmptyInstrument successful, Duration 0.29 seconds\nMaskBTP-[Notice] MaskBTP successful, Duration 0.47 seconds\nMaskBTP-[Notice] MaskBTP started\nLoadEmptyInstrument-[Notice] LoadEmptyInstrument started (child)\nLoadEmptyInstrument-[Notice] LoadEmptyInstrument successful, Duration 0.36 seconds\nMaskBTP-[Notice] MaskBTP successful, Duration 0.61 seconds\n.MaskBTP-[Notice] MaskBTP started\nLoadEmptyInstrument-[Notice] LoadEmptyInstrument started (child)\nLoadEmptyInstrument-[Notice] LoadEmptyInstrument successful, Duration 0.67 seconds\nMaskBTP-[Notice] MaskBTP successful, Duration 0.77 seconds\n.LoadEmptyInstrument-[Notice] LoadEmptyInstrument started\nLoadEmptyInstrument-[Notice] LoadEmptyInstrument successful, Duration 0.16 seconds\nMaskBTP-[Notice] MaskBTP started\nMaskBTP-[Notice] MaskBTP successful, Duration 0.15 seconds\n.LoadEmptyInstrument-[Notice] LoadEmptyInstrument started\nLoadEmptyInstrument-[Notice] LoadEmptyInstrument successful, Duration 0.35 seconds\nMaskBTP-[Notice] MaskBTP started\nMaskBTP-[Notice] MaskBTP successful, Duration 0.12 seconds\n.LoadEmptyInstrument-[Notice] LoadEmptyInstrument started\nLoadEmptyInstrument-[Notice] LoadEmptyInstrument successful, Duration 0.36 seconds\nMaskBTP-[Notice] MaskBTP started\nMaskBTP-[Notice] MaskBTP successful, Duration 0.01 seconds\nLoadEmptyInstrument-[Notice] LoadEmptyInstrument started\nLoadEmptyInstrument-[Notice] LoadEmptyInstrument successful, Duration 0.07 seconds\nMaskBTP-[Notice] MaskBTP started\nMaskBTP-[Notice] MaskBTP successful, Duration 0.01 seconds\n.LoadEmptyInstrument-[Notice] LoadEmptyInstrument started\nLoadEmptyInstrument-[Notice] LoadEmptyInstrument successful, Duration 0.72 seconds\nMaskBTP-[Notice] MaskBTP started\nMaskBTP-[Notice] MaskBTP successful, Duration 0.05 seconds\n.LoadEmptyInstrument-[Notice] LoadEmptyInstrument started\nLoadEmptyInstrument-[Notice] LoadEmptyInstrument successful, Duration 0.18 seconds\nMaskBTP-[Notice] MaskBTP started\nMaskBTP-[Notice] MaskBTP successful, Duration 0.02 seconds\n.LoadEmptyInstrument-[Notice] LoadEmptyInstrument started\nLoadEmptyInstrument-[Notice] LoadEmptyInstrument successful, Duration 0.92 seconds\nMaskBTP-[Notice] MaskBTP started\nMaskBTP-[Notice] MaskBTP successful, Duration 0.06 seconds\n.LoadEmptyInstrument-[Notice] LoadEmptyInstrument started\nLoadEmptyInstrument-[Notice] LoadEmptyInstrument successful, Duration 0.92 seconds\nMaskBTP-[Notice] MaskBTP started\nMaskBTP-[Notice] MaskBTP successful, Duration 0.16 seconds\n.LoadEmptyInstrument-[Notice] LoadEmptyInstrument started\nLoadEmptyInstrument-[Notice] LoadEmptyInstrument successful, Duration 0.33 seconds\nMaskBTP-[Notice] MaskBTP started\nMaskBTP-[Notice] MaskBTP successful, Duration 0.02 seconds\n.LoadEmptyInstrument-[Notice] LoadEmptyInstrument started\nLoadEmptyInstrument-[Notice] LoadEmptyInstrument successful, Duration 0.68 seconds\nMaskBTP-[Notice] MaskBTP started\nMaskBTP-[Notice] MaskBTP successful, Duration 0.20 seconds\n.LoadEmptyInstrument-[Notice] LoadEmptyInstrument started\nLoadEmptyInstrument-[Notice] LoadEmptyInstrument successful, Duration 0.08 seconds\nMaskBTP-[Notice] MaskBTP started\nMaskBTP-[Notice] MaskBTP successful, Duration 0.01 seconds\n.LoadEmptyInstrument-[Notice] LoadEmptyInstrument started\nLoadEmptyInstrument-[Notice] LoadEmptyInstrument successful, Duration 0.79 seconds\nMaskBTP-[Notice] MaskBTP started\nMaskBTP-[Notice] MaskBTP successful, Duration 0.19 seconds\nMaskBTP-[Notice] MaskBTP started\nMaskBTP-[Notice] MaskBTP successful, Duration 0.06 seconds\n.LoadEmptyInstrument-[Notice] LoadEmptyInstrument started\nLoadEmptyInstrument-[Notice] LoadEmptyInstrument successful, Duration 0.73 seconds\nMaskBTP-[Notice] MaskBTP started\nMaskBTP-[Notice] MaskBTP successful, Duration 0.20 seconds\n.LoadEmptyInstrument-[Notice] LoadEmptyInstrument started\nLoadEmptyInstrument-[Notice] LoadEmptyInstrument successful, Duration 0.15 seconds\nMaskBTP-[Notice] MaskBTP started\nMaskBTP-[Notice] MaskBTP successful, Duration 0.14 seconds\n.\n======================================================================\nFAIL: testEdges (MaskBTPTest.MaskBTPTest)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File &quot;/jenkins_workdir/workspace/pull_requests-conda-linux/Framework/PythonInterface/test/python/plugins/algorithms/MaskBTPTest.py&quot;, line 152, in testEdges\n    self.assertEqual(4 * 24, len(masking))\nAssertionError: 96 != 100\n\n----------------------------------------------------------------------\nRan 21 tests in 37.495s\n\nFAILED (failures=1)\n\n          Start 1696: python.WorkflowAlgorithms.WANDPowderReductionTest.WANDPowderReductionTest\n"
            }
        },
        {
            "test_name": "python.MuonQt5.basic_fitting_view_test.basic_fitting_view_test",
            "latest_failed_build": "1971",
            "agent_name": "Linux",
            "last_failed_outcome": "Flake",
            "pipeline_name": "pull_requests-conda-linux",
            "problem_count": 1,
            "past_outcome": {
                "build_keys": [
                    "1991",
                    "1990",
                    "1989",
                    "1988",
                    "1987",
                    "1986",
                    "1985",
                    "1984",
                    "1983",
                    "1982",
                    "1981",
                    "1980",
                    "1979",
                    "1978",
                    "1977",
                    "1976",
                    "1975",
                    "1974",
                    "1973",
                    "1972",
                    "1971",
                    "1970",
                    "1969",
                    "1968",
                    "1967",
                    "1966",
                    "1965",
                    "1964",
                    "1963",
                    "1962"
                ],
                "past_outcome_list": [
                    "None",
                    "None",
                    "Passed",
                    "None",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "None",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Flake",
                    "None",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed"
                ]
            },
            "last_stack_traces": {
                "test_name": "python.MuonQt5.basic_fitting_view_test.basic_fitting_view_test",
                "build": "1971",
                "agent_name": "Linux",
                "trial": 1,
                "outcome": "Failed",
                "test_time": "3.88",
                "stacktrace": "FrameworkManager-[Notice] Welcome to Mantid 6.5.20221121.1541\nFrameworkManager-[Notice] Please cite: http://dx.doi.org/10.1016/j.nima.2014.07.029 and this release: http://dx.doi.org/10.5286/Software/Mantid\nQStandardPaths: XDG_RUNTIME_DIR not set, defaulting to &#x27;/tmp/runtime-abc&#x27;\nlibEGL warning: DRI2: failed to open swrast (search paths /usr/lib64/dri)\nlibEGL warning: DRI2: failed to open swrast (search paths /usr/lib64/dri)\nlibEGL warning: DRI2: failed to open swrast (search paths /usr/lib64/dri)\nlibEGL warning: DRI2: failed to open swrast (search paths /usr/lib64/dri)\nQXcbIntegration: Cannot create platform OpenGL context, neither GLX nor EGL are enabled\nEEEEEEEEEEEEEEEEEEEEEEEEEE\n======================================================================\nERROR: test_that_enable_view_does_enable_the_tab_if_there_are_datasets_loaded (basic_fitting_view_test.BasicFittingViewTest)\n----------------------------------------------------------------------\nAttributeError: &#x27;BasicFittingView&#x27; object has no attribute &#x27;y&#x27;\n\nThe above exception was the direct cause of the following exception:\n\nTraceback (most recent call last):\n  File &quot;/jenkins_workdir/workspace/pull_requests-conda-linux/qt/python/mantidqtinterfaces/test/Muon/fitting_widgets/basic_fitting/basic_fitting_view_test.py&quot;, line 32, in setUp\n    self.view = BasicFittingView()\n  File &quot;/jenkins_workdir/workspace/pull_requests-conda-linux/qt/python/mantidqtinterfaces/mantidqtinterfaces/Muon/GUI/Common/fitting_widgets/basic_fitting/basic_fitting_view.py&quot;, line 29, in __init__\n    self.setupUi(self)\n  File &quot;&lt;string&gt;&quot;, line 31, in setupUi\nSystemError: &lt;built-in function connectSlotsByName&gt; returned a result with an error set\n\n======================================================================\nERROR: test_that_enable_view_does_not_enable_the_tab_if_there_are_no_datasets_loaded (basic_fitting_view_test.BasicFittingViewTest)\n----------------------------------------------------------------------\nAttributeError: &#x27;BasicFittingView&#x27; object has no attribute &#x27;y&#x27;\n\nThe above exception was the direct cause of the following exception:\n\nTraceback (most recent call last):\n  File &quot;/jenkins_workdir/workspace/pull_requests-conda-linux/qt/python/mantidqtinterfaces/test/Muon/fitting_widgets/basic_fitting/basic_fitting_view_test.py&quot;, line 32, in setUp\n    self.view = BasicFittingView()\n  File &quot;/jenkins_workdir/workspace/pull_requests-conda-linux/qt/python/mantidqtinterfaces/mantidqtinterfaces/Muon/GUI/Common/fitting_widgets/basic_fitting/basic_fitting_view.py&quot;, line 29, in __init__\n    self.setupUi(self)\n  File &quot;&lt;string&gt;&quot;, line 31, in setupUi\nSystemError: &lt;built-in function connectSlotsByName&gt; returned a result with an error set\n\n======================================================================\nERROR: test_that_hide_evaluate_function_as_checkbox_will_hide_the_evaluate_function_table_row (basic_fitting_view_test.BasicFittingViewTest)\n----------------------------------------------------------------------\nAttributeError: &#x27;BasicFittingView&#x27; object has no attribute &#x27;y&#x27;\n\nThe above exception was the direct cause of the following exception:\n\nTraceback (most recent call last):\n  File &quot;/jenkins_workdir/workspace/pull_requests-conda-linux/qt/python/mantidqtinterfaces/test/Muon/fitting_widgets/basic_fitting/basic_fitting_view_test.py&quot;, line 32, in setUp\n    self.view = BasicFittingView()\n  File &quot;/jenkins_workdir/workspace/pull_requests-conda-linux/qt/python/mantidqtinterfaces/mantidqtinterfaces/Muon/GUI/Common/fitting_widgets/basic_fitting/basic_fitting_view.py&quot;, line 29, in __init__\n    self.setupUi(self)\n  File &quot;&lt;string&gt;&quot;, line 31, in setupUi\nSystemError: &lt;built-in function connectSlotsByName&gt; returned a result with an error set\n\n======================================================================\nERROR: test_that_hide_exclude_range_checkbox_will_hide_the_exclude_range_table_row (basic_fitting_view_test.BasicFittingViewTest)\n----------------------------------------------------------------------\nAttributeError: &#x27;BasicFittingView&#x27; object has no attribute &#x27;y&#x27;\n\nThe above exception was the direct cause of the following exception:\n\nTraceback (most recent call last):\n  File &quot;/jenkins_workdir/workspace/pull_requests-conda-linux/qt/python/mantidqtinterfaces/test/Muon/fitting_widgets/basic_fitting/basic_fitting_view_test.py&quot;, line 32, in setUp\n    self.view = BasicFittingView()\n  File &quot;/jenkins_workdir/workspace/pull_requests-conda-linux/qt/python/mantidqtinterfaces/mantidqtinterfaces/Muon/GUI/Common/fitting_widgets/basic_fitting/basic_fitting_view.py&quot;, line 29, in __init__\n    self.setupUi(self)\n  File &quot;&lt;string&gt;&quot;, line 31, in setupUi\nSystemError: &lt;built-in function connectSlotsByName&gt; returned a result with an error set\n\n======================================================================\nERROR: test_that_it_is_possible_to_set_the_end_x_to_a_different_value (basic_fitting_view_test.BasicFittingViewTest)\n----------------------------------------------------------------------\nAttributeError: &#x27;BasicFittingView&#x27; object has no attribute &#x27;y&#x27;\n\nThe above exception was the direct cause of the following exception:\n\nTraceback (most recent call last):\n  File &quot;/jenkins_workdir/workspace/pull_requests-conda-linux/qt/python/mantidqtinterfaces/test/Muon/fitting_widgets/basic_fitting/basic_fitting_view_test.py&quot;, line 32, in setUp\n    self.view = BasicFittingView()\n  File &quot;/jenkins_workdir/workspace/pull_requests-conda-linux/qt/python/mantidqtinterfaces/mantidqtinterfaces/Muon/GUI/Common/fitting_widgets/basic_fitting/basic_fitting_view.py&quot;, line 29, in __init__\n    self.setupUi(self)\n  File &quot;&lt;string&gt;&quot;, line 31, in setupUi\nSystemError: &lt;built-in function connectSlotsByName&gt; returned a result with an error set\n\n======================================================================\nERROR: test_that_it_is_possible_to_set_the_start_x_to_a_different_value (basic_fitting_view_test.BasicFittingViewTest)\n----------------------------------------------------------------------\nAttributeError: &#x27;BasicFittingView&#x27; object has no attribute &#x27;y&#x27;\n\nThe above exception was the direct cause of the following exception:\n\nTraceback (most recent call last):\n  File &quot;/jenkins_workdir/workspace/pull_requests-conda-linux/qt/python/mantidqtinterfaces/test/Muon/fitting_widgets/basic_fitting/basic_fitting_view_test.py&quot;, line 32, in setUp\n    self.view = BasicFittingView()\n  File &quot;/jenkins_workdir/workspace/pull_requests-conda-linux/qt/python/mantidqtinterfaces/mantidqtinterfaces/Muon/GUI/Common/fitting_widgets/basic_fitting/basic_fitting_view.py&quot;, line 29, in __init__\n    self.setupUi(self)\n  File &quot;&lt;string&gt;&quot;, line 31, in setupUi\nSystemError: &lt;built-in function connectSlotsByName&gt; returned a result with an error set\n\n======================================================================\nERROR: test_that_set_current_dataset_index_will_set_the_current_dataset_index_in_the_function_browser (basic_fitting_view_test.BasicFittingViewTest)\n----------------------------------------------------------------------\nAttributeError: &#x27;BasicFittingView&#x27; object has no attribute &#x27;y&#x27;\n\nThe above exception was the direct cause of the following exception:\n\nTraceback (most recent call last):\n  File &quot;/jenkins_workdir/workspace/pull_requests-conda-linux/qt/python/mantidqtinterfaces/test/Muon/fitting_widgets/basic_fitting/basic_fitting_view_test.py&quot;, line 32, in setUp\n    self.view = BasicFittingView()\n  File &quot;/jenkins_workdir/workspace/pull_requests-conda-linux/qt/python/mantidqtinterfaces/mantidqtinterfaces/Muon/GUI/Common/fitting_widgets/basic_fitting/basic_fitting_view.py&quot;, line 29, in __init__\n    self.setupUi(self)\n  File &quot;&lt;string&gt;&quot;, line 31, in setupUi\nSystemError: &lt;built-in function connectSlotsByName&gt; returned a result with an error set\n\n======================================================================\nERROR: test_that_set_datasets_in_function_browser_will_set_the_datasets_in_the_function_browser (basic_fitting_view_test.BasicFittingViewTest)\n----------------------------------------------------------------------\nAttributeError: &#x27;BasicFittingView&#x27; object has no attribute &#x27;y&#x27;\n\nThe above exception was the direct cause of the following exception:\n\nTraceback (most recent call last):\n  File &quot;/jenkins_workdir/workspace/pull_requests-conda-linux/qt/python/mantidqtinterfaces/test/Muon/fitting_widgets/basic_fitting/basic_fitting_view_test.py&quot;, line 32, in setUp\n    self.view = BasicFittingView()\n  File &quot;/jenkins_workdir/workspace/pull_requests-conda-linux/qt/python/mantidqtinterfaces/mantidqtinterfaces/Muon/GUI/Common/fitting_widgets/basic_fitting/basic_fitting_view.py&quot;, line 29, in __init__\n    self.setupUi(self)\n  File &quot;&lt;string&gt;&quot;, line 31, in setupUi\nSystemError: &lt;built-in function connectSlotsByName&gt; returned a result with an error set\n\n======================================================================\nERROR: test_that_show_plot_guess_end_x_will_show_plot_guess_end_x_row (basic_fitting_view_test.BasicFittingViewTest)\n----------------------------------------------------------------------\nAttributeError: &#x27;BasicFittingView&#x27; object has no attribute &#x27;y&#x27;\n\nThe above exception was the direct cause of the following exception:\n\nTraceback (most recent call last):\n  File &quot;/jenkins_workdir/workspace/pull_requests-conda-linux/qt/python/mantidqtinterfaces/test/Muon/fitting_widgets/basic_fitting/basic_fitting_view_test.py&quot;, line 32, in setUp\n    self.view = BasicFittingView()\n  File &quot;/jenkins_workdir/workspace/pull_requests-conda-linux/qt/python/mantidqtinterfaces/mantidqtinterfaces/Muon/GUI/Common/fitting_widgets/basic_fitting/basic_fitting_view.py&quot;, line 29, in __init__\n    self.setupUi(self)\n  File &quot;&lt;string&gt;&quot;, line 31, in setupUi\nSystemError: &lt;built-in function connectSlotsByName&gt; returned a result with an error set\n\n======================================================================\nERROR: test_that_show_plot_guess_points_will_show_plot_guess_points_row (basic_fitting_view_test.BasicFittingViewTest)\n----------------------------------------------------------------------\nAttributeError: &#x27;BasicFittingView&#x27; object has no attribute &#x27;y&#x27;\n\nThe above exception was the direct cause of the following exception:\n\nTraceback (most recent call last):\n  File &quot;/jenkins_workdir/workspace/pull_requests-conda-linux/qt/python/mantidqtinterfaces/test/Muon/fitting_widgets/basic_fitting/basic_fitting_view_test.py&quot;, line 32, in setUp\n    self.view = BasicFittingView()\n  File &quot;/jenkins_workdir/workspace/pull_requests-conda-linux/qt/python/mantidqtinterfaces/mantidqtinterfaces/Muon/GUI/Common/fitting_widgets/basic_fitting/basic_fitting_view.py&quot;, line 29, in __init__\n    self.setupUi(self)\n  File &quot;&lt;string&gt;&quot;, line 31, in setupUi\nSystemError: &lt;built-in function connectSlotsByName&gt; returned a result with an error set\n\n======================================================================\nERROR: test_that_show_plot_guess_start_x_will_show_plot_guess_start_x_row (basic_fitting_view_test.BasicFittingViewTest)\n----------------------------------------------------------------------\nAttributeError: &#x27;BasicFittingView&#x27; object has no attribute &#x27;y&#x27;\n\nThe above exception was the direct cause of the following exception:\n\nTraceback (most recent call last):\n  File &quot;/jenkins_workdir/workspace/pull_requests-conda-linux/qt/python/mantidqtinterfaces/test/Muon/fitting_widgets/basic_fitting/basic_fitting_view_test.py&quot;, line 32, in setUp\n    self.view = BasicFittingView()\n  File &quot;/jenkins_workdir/workspace/pull_requests-conda-linux/qt/python/mantidqtinterfaces/mantidqtinterfaces/Muon/GUI/Common/fitting_widgets/basic_fitting/basic_fitting_view.py&quot;, line 29, in __init__\n    self.setupUi(self)\n  File &quot;&lt;string&gt;&quot;, line 31, in setupUi\nSystemError: &lt;built-in function connectSlotsByName&gt; returned a result with an error set\n\n======================================================================\nERROR: test_that_the_fit_to_raw_checkbox_value_can_be_changed_as_expected (basic_fitting_view_test.BasicFittingViewTest)\n----------------------------------------------------------------------\nAttributeError: &#x27;BasicFittingView&#x27; object has no attribute &#x27;y&#x27;\n\nThe above exception was the direct cause of the following exception:\n\nTraceback (most recent call last):\n  File &quot;/jenkins_workdir/workspace/pull_requests-conda-linux/qt/python/mantidqtinterfaces/test/Muon/fitting_widgets/basic_fitting/basic_fitting_view_test.py&quot;, line 32, in setUp\n    self.view = BasicFittingView()\n  File &quot;/jenkins_workdir/workspace/pull_requests-conda-linux/qt/python/mantidqtinterfaces/mantidqtinterfaces/Muon/GUI/Common/fitting_widgets/basic_fitting/basic_fitting_view.py&quot;, line 29, in __init__\n    self.setupUi(self)\n  File &quot;&lt;string&gt;&quot;, line 31, in setupUi\nSystemError: &lt;built-in function connectSlotsByName&gt; returned a result with an error set\n\n======================================================================\nERROR: test_that_the_function_name_can_be_changed_as_expected (basic_fitting_view_test.BasicFittingViewTest)\n----------------------------------------------------------------------\nAttributeError: &#x27;BasicFittingView&#x27; object has no attribute &#x27;y&#x27;\n\nThe above exception was the direct cause of the following exception:\n\nTraceback (most recent call last):\n  File &quot;/jenkins_workdir/workspace/pull_requests-conda-linux/qt/python/mantidqtinterfaces/test/Muon/fitting_widgets/basic_fitting/basic_fitting_view_test.py&quot;, line 32, in setUp\n    self.view = BasicFittingView()\n  File &quot;/jenkins_workdir/workspace/pull_requests-conda-linux/qt/python/mantidqtinterfaces/mantidqtinterfaces/Muon/GUI/Common/fitting_widgets/basic_fitting/basic_fitting_view.py&quot;, line 29, in __init__\n    self.setupUi(self)\n  File &quot;&lt;string&gt;&quot;, line 31, in setupUi\nSystemError: &lt;built-in function connectSlotsByName&gt; returned a result with an error set\n\n======================================================================\nERROR: test_that_the_plot_guess_checkbox_can_be_ticked_as_expected (basic_fitting_view_test.BasicFittingViewTest)\n----------------------------------------------------------------------\nAttributeError: &#x27;BasicFittingView&#x27; object has no attribute &#x27;y&#x27;\n\nThe above exception was the direct cause of the following exception:\n\nTraceback (most recent call last):\n  File &quot;/jenkins_workdir/workspace/pull_requests-conda-linux/qt/python/mantidqtinterfaces/test/Muon/fitting_widgets/basic_fitting/basic_fitting_view_test.py&quot;, line 32, in setUp\n    self.view = BasicFittingView()\n  File &quot;/jenkins_workdir/workspace/pull_requests-conda-linux/qt/python/mantidqtinterfaces/mantidqtinterfaces/Muon/GUI/Common/fitting_widgets/basic_fitting/basic_fitting_view.py&quot;, line 29, in __init__\n    self.setupUi(self)\n  File &quot;&lt;string&gt;&quot;, line 31, in setupUi\nSystemError: &lt;built-in function connectSlotsByName&gt; returned a result with an error set\n\n======================================================================\nERROR: test_that_the_undo_fit_button_can_be_enabled_as_expected_when_the_number_of_undos_is_above_zero (basic_fitting_view_test.BasicFittingViewTest)\n----------------------------------------------------------------------\nAttributeError: &#x27;BasicFittingView&#x27; object has no attribute &#x27;y&#x27;\n\nThe above exception was the direct cause of the following exception:\n\nTraceback (most recent call last):\n  File &quot;/jenkins_workdir/workspace/pull_requests-conda-linux/qt/python/mantidqtinterfaces/test/Muon/fitting_widgets/basic_fitting/basic_fitting_view_test.py&quot;, line 32, in setUp\n    self.view = BasicFittingView()\n  File &quot;/jenkins_workdir/workspace/pull_requests-conda-linux/qt/python/mantidqtinterfaces/mantidqtinterfaces/Muon/GUI/Common/fitting_widgets/basic_fitting/basic_fitting_view.py&quot;, line 29, in __init__\n    self.setupUi(self)\n  File &quot;&lt;string&gt;&quot;, line 31, in setupUi\nSystemError: &lt;built-in function connectSlotsByName&gt; returned a result with an error set\n\n======================================================================\nERROR: test_that_the_view_has_been_initialized_with_the_exclude_range_start_and_end_x_options_not_visible (basic_fitting_view_test.BasicFittingViewTest)\n----------------------------------------------------------------------\nAttributeError: &#x27;BasicFittingView&#x27; object has no attribute &#x27;y&#x27;\n\nThe above exception was the direct cause of the following exception:\n\nTraceback (most recent call last):\n  File &quot;/jenkins_workdir/workspace/pull_requests-conda-linux/qt/python/mantidqtinterfaces/test/Muon/fitting_widgets/basic_fitting/basic_fitting_view_test.py&quot;, line 32, in setUp\n    self.view = BasicFittingView()\n  File &quot;/jenkins_workdir/workspace/pull_requests-conda-linux/qt/python/mantidqtinterfaces/mantidqtinterfaces/Muon/GUI/Common/fitting_widgets/basic_fitting/basic_fitting_view.py&quot;, line 29, in __init__\n    self.setupUi(self)\n  File &quot;&lt;string&gt;&quot;, line 31, in setupUi\nSystemError: &lt;built-in function connectSlotsByName&gt; returned a result with an error set\n\n======================================================================\nERROR: test_that_the_view_has_been_initialized_with_the_raw_data_option_hidden_when_it_is_a_frequency_domain (basic_fitting_view_test.BasicFittingViewTest)\n----------------------------------------------------------------------\nAttributeError: &#x27;BasicFittingView&#x27; object has no attribute &#x27;y&#x27;\n\nThe above exception was the direct cause of the following exception:\n\nTraceback (most recent call last):\n  File &quot;/jenkins_workdir/workspace/pull_requests-conda-linux/qt/python/mantidqtinterfaces/test/Muon/fitting_widgets/basic_fitting/basic_fitting_view_test.py&quot;, line 32, in setUp\n    self.view = BasicFittingView()\n  File &quot;/jenkins_workdir/workspace/pull_requests-conda-linux/qt/python/mantidqtinterfaces/mantidqtinterfaces/Muon/GUI/Common/fitting_widgets/basic_fitting/basic_fitting_view.py&quot;, line 29, in __init__\n    self.setupUi(self)\n  File &quot;&lt;string&gt;&quot;, line 31, in setupUi\nSystemError: &lt;built-in function connectSlotsByName&gt; returned a result with an error set\n\n======================================================================\nERROR: test_that_the_view_has_been_initialized_with_the_raw_data_option_shown_when_it_is_not_a_frequency_domain (basic_fitting_view_test.BasicFittingViewTest)\n----------------------------------------------------------------------\nAttributeError: &#x27;BasicFittingView&#x27; object has no attribute &#x27;y&#x27;\n\nThe above exception was the direct cause of the following exception:\n\nTraceback (most recent call last):\n  File &quot;/jenkins_workdir/workspace/pull_requests-conda-linux/qt/python/mantidqtinterfaces/test/Muon/fitting_widgets/basic_fitting/basic_fitting_view_test.py&quot;, line 32, in setUp\n    self.view = BasicFittingView()\n  File &quot;/jenkins_workdir/workspace/pull_requests-conda-linux/qt/python/mantidqtinterfaces/mantidqtinterfaces/Muon/GUI/Common/fitting_widgets/basic_fitting/basic_fitting_view.py&quot;, line 29, in __init__\n    self.setupUi(self)\n  File &quot;&lt;string&gt;&quot;, line 31, in setupUi\nSystemError: &lt;built-in function connectSlotsByName&gt; returned a result with an error set\n\n======================================================================\nERROR: test_that_update_fit_status_labels_will_display_fit_successful_if_all_fits_are_successful (basic_fitting_view_test.BasicFittingViewTest)\n----------------------------------------------------------------------\nAttributeError: &#x27;BasicFittingView&#x27; object has no attribute &#x27;y&#x27;\n\nThe above exception was the direct cause of the following exception:\n\nTraceback (most recent call last):\n  File &quot;/jenkins_workdir/workspace/pull_requests-conda-linux/qt/python/mantidqtinterfaces/test/Muon/fitting_widgets/basic_fitting/basic_fitting_view_test.py&quot;, line 32, in setUp\n    self.view = BasicFittingView()\n  File &quot;/jenkins_workdir/workspace/pull_requests-conda-linux/qt/python/mantidqtinterfaces/mantidqtinterfaces/Muon/GUI/Common/fitting_widgets/basic_fitting/basic_fitting_view.py&quot;, line 29, in __init__\n    self.setupUi(self)\n  File &quot;&lt;string&gt;&quot;, line 31, in setupUi\nSystemError: &lt;built-in function connectSlotsByName&gt; returned a result with an error set\n\n======================================================================\nERROR: test_that_update_fit_status_labels_will_display_fits_failed_if_some_of_the_fits_fail (basic_fitting_view_test.BasicFittingViewTest)\n----------------------------------------------------------------------\nAttributeError: &#x27;BasicFittingView&#x27; object has no attribute &#x27;y&#x27;\n\nThe above exception was the direct cause of the following exception:\n\nTraceback (most recent call last):\n  File &quot;/jenkins_workdir/workspace/pull_requests-conda-linux/qt/python/mantidqtinterfaces/test/Muon/fitting_widgets/basic_fitting/basic_fitting_view_test.py&quot;, line 32, in setUp\n    self.view = BasicFittingView()\n  File &quot;/jenkins_workdir/workspace/pull_requests-conda-linux/qt/python/mantidqtinterfaces/mantidqtinterfaces/Muon/GUI/Common/fitting_widgets/basic_fitting/basic_fitting_view.py&quot;, line 29, in __init__\n    self.setupUi(self)\n  File &quot;&lt;string&gt;&quot;, line 31, in setupUi\nSystemError: &lt;built-in function connectSlotsByName&gt; returned a result with an error set\n\n======================================================================\nERROR: test_that_update_fit_status_labels_will_display_no_fit_if_the_success_list_is_empty (basic_fitting_view_test.BasicFittingViewTest)\n----------------------------------------------------------------------\nAttributeError: &#x27;BasicFittingView&#x27; object has no attribute &#x27;y&#x27;\n\nThe above exception was the direct cause of the following exception:\n\nTraceback (most recent call last):\n  File &quot;/jenkins_workdir/workspace/pull_requests-conda-linux/qt/python/mantidqtinterfaces/test/Muon/fitting_widgets/basic_fitting/basic_fitting_view_test.py&quot;, line 32, in setUp\n    self.view = BasicFittingView()\n  File &quot;/jenkins_workdir/workspace/pull_requests-conda-linux/qt/python/mantidqtinterfaces/mantidqtinterfaces/Muon/GUI/Common/fitting_widgets/basic_fitting/basic_fitting_view.py&quot;, line 29, in __init__\n    self.setupUi(self)\n  File &quot;&lt;string&gt;&quot;, line 31, in setupUi\nSystemError: &lt;built-in function connectSlotsByName&gt; returned a result with an error set\n\n======================================================================\nERROR: test_that_update_function_browser_parameters_will_clear_the_function_if_the_function_provided_is_none (basic_fitting_view_test.BasicFittingViewTest)\n----------------------------------------------------------------------\nAttributeError: &#x27;BasicFittingView&#x27; object has no attribute &#x27;y&#x27;\n\nThe above exception was the direct cause of the following exception:\n\nTraceback (most recent call last):\n  File &quot;/jenkins_workdir/workspace/pull_requests-conda-linux/qt/python/mantidqtinterfaces/test/Muon/fitting_widgets/basic_fitting/basic_fitting_view_test.py&quot;, line 32, in setUp\n    self.view = BasicFittingView()\n  File &quot;/jenkins_workdir/workspace/pull_requests-conda-linux/qt/python/mantidqtinterfaces/mantidqtinterfaces/Muon/GUI/Common/fitting_widgets/basic_fitting/basic_fitting_view.py&quot;, line 29, in __init__\n    self.setupUi(self)\n  File &quot;&lt;string&gt;&quot;, line 31, in setupUi\nSystemError: &lt;built-in function connectSlotsByName&gt; returned a result with an error set\n\n======================================================================\nERROR: test_that_update_function_browser_parameters_will_update_the_parameters_of_the_function_for_single_fit (basic_fitting_view_test.BasicFittingViewTest)\n----------------------------------------------------------------------\nAttributeError: &#x27;BasicFittingView&#x27; object has no attribute &#x27;y&#x27;\n\nThe above exception was the direct cause of the following exception:\n\nTraceback (most recent call last):\n  File &quot;/jenkins_workdir/workspace/pull_requests-conda-linux/qt/python/mantidqtinterfaces/test/Muon/fitting_widgets/basic_fitting/basic_fitting_view_test.py&quot;, line 32, in setUp\n    self.view = BasicFittingView()\n  File &quot;/jenkins_workdir/workspace/pull_requests-conda-linux/qt/python/mantidqtinterfaces/mantidqtinterfaces/Muon/GUI/Common/fitting_widgets/basic_fitting/basic_fitting_view.py&quot;, line 29, in __init__\n    self.setupUi(self)\n  File &quot;&lt;string&gt;&quot;, line 31, in setupUi\nSystemError: &lt;built-in function connectSlotsByName&gt; returned a result with an error set\n\n======================================================================\nERROR: test_that_update_global_fit_status_label_will_display_fit_successful_if_all_fits_are_successful (basic_fitting_view_test.BasicFittingViewTest)\n----------------------------------------------------------------------\nAttributeError: &#x27;BasicFittingView&#x27; object has no attribute &#x27;y&#x27;\n\nThe above exception was the direct cause of the following exception:\n\nTraceback (most recent call last):\n  File &quot;/jenkins_workdir/workspace/pull_requests-conda-linux/qt/python/mantidqtinterfaces/test/Muon/fitting_widgets/basic_fitting/basic_fitting_view_test.py&quot;, line 32, in setUp\n    self.view = BasicFittingView()\n  File &quot;/jenkins_workdir/workspace/pull_requests-conda-linux/qt/python/mantidqtinterfaces/mantidqtinterfaces/Muon/GUI/Common/fitting_widgets/basic_fitting/basic_fitting_view.py&quot;, line 29, in __init__\n    self.setupUi(self)\n  File &quot;&lt;string&gt;&quot;, line 31, in setupUi\nSystemError: &lt;built-in function connectSlotsByName&gt; returned a result with an error set\n\n======================================================================\nERROR: test_that_update_global_fit_status_label_will_display_fits_failed_if_some_of_the_fits_fail (basic_fitting_view_test.BasicFittingViewTest)\n----------------------------------------------------------------------\nAttributeError: &#x27;BasicFittingView&#x27; object has no attribute &#x27;y&#x27;\n\nThe above exception was the direct cause of the following exception:\n\nTraceback (most recent call last):\n  File &quot;/jenkins_workdir/workspace/pull_requests-conda-linux/qt/python/mantidqtinterfaces/test/Muon/fitting_widgets/basic_fitting/basic_fitting_view_test.py&quot;, line 32, in setUp\n    self.view = BasicFittingView()\n  File &quot;/jenkins_workdir/workspace/pull_requests-conda-linux/qt/python/mantidqtinterfaces/mantidqtinterfaces/Muon/GUI/Common/fitting_widgets/basic_fitting/basic_fitting_view.py&quot;, line 29, in __init__\n    self.setupUi(self)\n  File &quot;&lt;string&gt;&quot;, line 31, in setupUi\nSystemError: &lt;built-in function connectSlotsByName&gt; returned a result with an error set\n\n======================================================================\nERROR: test_that_update_global_fit_status_label_will_display_no_fit_if_the_success_list_is_empty (basic_fitting_view_test.BasicFittingViewTest)\n----------------------------------------------------------------------\nAttributeError: &#x27;BasicFittingView&#x27; object has no attribute &#x27;y&#x27;\n\nThe above exception was the direct cause of the following exception:\n\nTraceback (most recent call last):\n  File &quot;/jenkins_workdir/workspace/pull_requests-conda-linux/qt/python/mantidqtinterfaces/test/Muon/fitting_widgets/basic_fitting/basic_fitting_view_test.py&quot;, line 32, in setUp\n    self.view = BasicFittingView()\n  File &quot;/jenkins_workdir/workspace/pull_requests-conda-linux/qt/python/mantidqtinterfaces/mantidqtinterfaces/Muon/GUI/Common/fitting_widgets/basic_fitting/basic_fitting_view.py&quot;, line 29, in __init__\n    self.setupUi(self)\n  File &quot;&lt;string&gt;&quot;, line 31, in setupUi\nSystemError: &lt;built-in function connectSlotsByName&gt; returned a result with an error set\n\n----------------------------------------------------------------------\nRan 26 tests in 0.173s\n\nFAILED (errors=26)\n\n          Start 2178: python.MuonQt5.basic_fitting_view_test.basic_fitting_view_test\n"
            }
        },
        {
            "test_name": "python.EngineeringDiffraction.test_gsas2_model.test_gsas2_model",
            "latest_failed_build": "1965",
            "agent_name": "Linux",
            "last_failed_outcome": "Failed",
            "pipeline_name": "pull_requests-conda-linux",
            "problem_count": 2,
            "past_outcome": {
                "build_keys": [
                    "1991",
                    "1990",
                    "1989",
                    "1988",
                    "1987",
                    "1986",
                    "1985",
                    "1984",
                    "1983",
                    "1982",
                    "1981",
                    "1980",
                    "1979",
                    "1978",
                    "1977",
                    "1976",
                    "1975",
                    "1974",
                    "1973",
                    "1972",
                    "1971",
                    "1970",
                    "1969",
                    "1968",
                    "1967",
                    "1966",
                    "1965",
                    "1964",
                    "1963",
                    "1962"
                ],
                "past_outcome_list": [
                    "None",
                    "None",
                    "Passed",
                    "None",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "None",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "None",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Failed",
                    "Passed",
                    "Failed",
                    "Passed"
                ]
            },
            "last_stack_traces": {
                "test_name": "python.EngineeringDiffraction.test_gsas2_model.test_gsas2_model",
                "build": "1965",
                "agent_name": "Linux",
                "trial": 3,
                "outcome": "Failed",
                "test_time": "6.02",
                "stacktrace": "FrameworkManager-[Notice] Welcome to Mantid 6.5.20221118.1650\nFrameworkManager-[Notice] Please cite: http://dx.doi.org/10.1016/j.nima.2014.07.029 and this release: http://dx.doi.org/10.5286/Software/Mantid\n...Python-[Error] GSAS-II call must have failed, as the output project file was not found.\n\n\n---------------------\n---------------------\n Errors from GSAS-II \n---------------------\n---------------------\nDetailed\nError\nMessage\n---------------------\n---------------------\n\n\n\n.Python-[Error] Invalid Phase file format in invalid_phase_file\n.CreateEmptyTableWorkspace-[Notice] CreateEmptyTableWorkspace started\nCreateEmptyTableWorkspace-[Notice] CreateEmptyTableWorkspace successful, Duration 0.00 seconds\nCreateEmptyTableWorkspace-[Notice] CreateEmptyTableWorkspace started\nCreateEmptyTableWorkspace-[Notice] CreateEmptyTableWorkspace successful, Duration 0.00 seconds\nCompareWorkspaces-[Notice] CompareWorkspaces started\nCompareWorkspaces-[Notice] The workspaces &quot;gsas2_output_GSASII_instrument_parameters&quot; and &quot;expected_gsas2_output_GSASII_instrument_parameters&quot; matched!\nCompareWorkspaces-[Notice] CompareWorkspaces successful, Duration 0.00 seconds\nCreateEmptyTableWorkspace-[Notice] CreateEmptyTableWorkspace started\nCreateEmptyTableWorkspace-[Notice] CreateEmptyTableWorkspace successful, Duration 0.00 seconds\nCreateEmptyTableWorkspace-[Notice] CreateEmptyTableWorkspace started\nCreateEmptyTableWorkspace-[Notice] CreateEmptyTableWorkspace successful, Duration 0.00 seconds\nCompareWorkspaces-[Notice] CompareWorkspaces started\nCompareWorkspaces-[Notice] The workspaces &quot;gsas2_output_GSASII_instrument_parameters&quot; and &quot;expected_gsas2_output_GSASII_instrument_parameters&quot; matched!\nCompareWorkspaces-[Notice] CompareWorkspaces successful, Duration 0.00 seconds\nCreateEmptyTableWorkspace-[Notice] CreateEmptyTableWorkspace started\nCreateEmptyTableWorkspace-[Notice] CreateEmptyTableWorkspace successful, Duration 0.00 seconds\nCreateEmptyTableWorkspace-[Notice] CreateEmptyTableWorkspace started\nCreateEmptyTableWorkspace-[Notice] CreateEmptyTableWorkspace successful, Duration 0.00 seconds\nCompareWorkspaces-[Notice] CompareWorkspaces started\nCompareWorkspaces-[Notice] The workspaces &quot;gsas2_output_GSASII_instrument_parameters&quot; and &quot;expected_gsas2_output_GSASII_instrument_parameters&quot; matched!\nCompareWorkspaces-[Notice] CompareWorkspaces successful, Duration 0.00 seconds\nCreateEmptyTableWorkspace-[Notice] CreateEmptyTableWorkspace started\nCreateEmptyTableWorkspace-[Notice] CreateEmptyTableWorkspace successful, Duration 0.00 seconds\nCreateEmptyTableWorkspace-[Notice] CreateEmptyTableWorkspace started\nCreateEmptyTableWorkspace-[Notice] CreateEmptyTableWorkspace successful, Duration 0.00 seconds\nCompareWorkspaces-[Notice] CompareWorkspaces started\nCompareWorkspaces-[Notice] The workspaces &quot;gsas2_output_GSASII_instrument_parameters&quot; and &quot;expected_gsas2_output_GSASII_instrument_parameters&quot; matched!\nCompareWorkspaces-[Notice] CompareWorkspaces successful, Duration 0.00 seconds\n.CreateEmptyTableWorkspace-[Notice] CreateEmptyTableWorkspace started\nCreateEmptyTableWorkspace-[Notice] CreateEmptyTableWorkspace successful, Duration 0.00 seconds\nCreateEmptyTableWorkspace-[Notice] CreateEmptyTableWorkspace started\nCreateEmptyTableWorkspace-[Notice] CreateEmptyTableWorkspace successful, Duration 0.00 seconds\nCompareWorkspaces-[Notice] CompareWorkspaces started\nCompareWorkspaces-[Notice] The workspaces &quot;gsas2_output_GSASII_lattice_parameters&quot; and &quot;expected_gsas2_output_GSASII_lattice_parameters&quot; matched!\nCompareWorkspaces-[Notice] CompareWorkspaces successful, Duration 0.00 seconds\n.CreateEmptyTableWorkspace-[Notice] CreateEmptyTableWorkspace started\nCreateEmptyTableWorkspace-[Notice] CreateEmptyTableWorkspace successful, Duration 0.00 seconds\nCreateEmptyTableWorkspace-[Notice] CreateEmptyTableWorkspace started\nCreateEmptyTableWorkspace-[Notice] CreateEmptyTableWorkspace successful, Duration 0.00 seconds\nCompareWorkspaces-[Notice] CompareWorkspaces started\nCompareWorkspaces-[Notice] The workspaces &quot;gsas2_output_GSASII_lattice_parameters&quot; and &quot;expected_gsas2_output_GSASII_lattice_parameters&quot; matched!\nCompareWorkspaces-[Notice] CompareWorkspaces successful, Duration 0.00 seconds\n..CreateEmptyTableWorkspace-[Notice] CreateEmptyTableWorkspace started\nCreateEmptyTableWorkspace-[Notice] CreateEmptyTableWorkspace successful, Duration 0.00 seconds\nCreateEmptyTableWorkspace-[Notice] CreateEmptyTableWorkspace started\nCreateEmptyTableWorkspace-[Notice] CreateEmptyTableWorkspace successful, Duration 0.00 seconds\nCompareWorkspaces-[Notice] CompareWorkspaces started\nCompareWorkspaces-[Notice] The workspaces &quot;gsas2_output_GSASII_reflections&quot; and &quot;expected_gsas2_output_GSASII_lattice_parameters&quot; matched!\nCompareWorkspaces-[Notice] CompareWorkspaces successful, Duration 0.00 seconds\n.Python-[Error] Different number of banks in file_mismatch and None\n.......Python-[Error] No Pawley Reflections were generated for the phases provided. Not calling GSAS-II.\n.Python-[Warning] Note the roto-inversion space group F m -3 m has been used rather than F m 3 m read from /jenkins_workdir/workspace/pull_requests-conda-linux/build/ExternalData/Testing/Data/UnitTest/EngDiff_gsas2_tab/gsas2_FE_GAMMA.cif as it is not in the accepted list of space groups.\n.Python-[Error] * All filepaths for Instrument, Phase and Focused Data must be valid\nPython-[Error] * There must be a valid Project Name for GSASII refinement\n...CreateWorkspace-[Notice] CreateWorkspace started\nCreateWorkspace-[Notice] CreateWorkspace successful, Duration 0.14 seconds\nReplaceSpecialValues-[Notice] ReplaceSpecialValues started\nReplaceSpecialValues-[Notice] ReplaceSpecialValues successful, Duration 0.01 seconds\nE..Python-[Error] Invalid Phase file format in invalid_phase_file\n.Python-[Notice] ENGINX_305761_307521_all_banks_TOF.gss\nPython-[Notice] Final refinement wR = 26.83%\n.Python-[Error] GSAS-II call must have failed, as the output project file was not found.\n\n\n---------------------\n---------------------\n Errors from GSAS-II \n---------------------\n---------------------\nErrors from GSAS-II\n---------------------\n---------------------\n\n\n\nPython-[Error] GSAS-II call must have failed, as the output result file was not found.\n\n\n---------------------\n---------------------\n Errors from GSAS-II \n---------------------\n---------------------\nErrors from GSAS-II\n---------------------\n---------------------\n\n\n\nPython-[Notice] \nGSAS-II call complete in 3.14 seconds.\n\n.LoadGSS-[Notice] LoadGSS started\nLoadGSS-[Warning] Line not defined: Monitor: 1\nLoadGSS-[Notice] LoadGSS successful, Duration 0.00 seconds\nDeleteWorkspace-[Notice] DeleteWorkspace started\nDeleteWorkspace-[Notice] DeleteWorkspace successful, Duration 0.00 seconds\n..Python-[Error] The number of instrument files (2) must be 1 or equal to the number of input histograms 3\n.\n======================================================================\nERROR: test_load_gsas_histogram (test_gsas2_model.TestGSAS2Model)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File &quot;/jenkins_workdir/workspace/pull_requests-conda-linux/Framework/PythonInterface/mantid/simpleapi.py&quot;, line 922, in do_set_property\n    alg_object.setProperty(key, new_value)\nValueError: Invalid value for property Filename (string) from string &quot;&quot;: No file specified.\n\nThe above exception was the direct cause of the following exception:\n\nTraceback (most recent call last):\n  File &quot;/jenkins_workdir/workspace/pull_requests-conda-linux/qt/python/mantidqtinterfaces/mantidqtinterfaces/Engineering/gui/engineering_diffraction/tabs/gsas2/test/test_gsas2_model.py&quot;, line 345, in test_load_gsas_histogram\n    expected_workspace = LoadNexus(FileFinder.getFullPath(&quot;gsas2_output_workspace_1.nxs&quot;))\n  File &quot;/jenkins_workdir/workspace/pull_requests-conda-linux/Framework/PythonInterface/mantid/simpleapi.py&quot;, line 1055, in __call__\n    set_properties(algm, *args, **final_keywords)\n  File &quot;/jenkins_workdir/workspace/pull_requests-conda-linux/Framework/PythonInterface/mantid/simpleapi.py&quot;, line 949, in set_properties\n    do_set_property(key, value)\n  File &quot;/jenkins_workdir/workspace/pull_requests-conda-linux/Framework/PythonInterface/mantid/simpleapi.py&quot;, line 926, in do_set_property\n    raise e.__class__(msg) from e\nValueError: Problem setting &quot;Filename&quot; in LoadNexus-v1: Invalid value for property Filename (string) from string &quot;&quot;: No file specified.\n\n----------------------------------------------------------------------\nRan 31 tests in 0.942s\n\nFAILED (errors=1)\n\n          Start  759: CurveFittingTest_ExpDecayMuonTest\n"
            }
        }
    ]
}